### PR TITLE
Jessachalla patch 1

### PR DESCRIPTION
**Pull request by jessachalla**
Creation! 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4121"></a>

### CODE REVIEW

**Comment by ChinmayMhatre** [comment]
Hey @jessachalla , Could you remove the file you added outside the data folder? Thank you!

File: data/jessachalla.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      "name": "I'm happy to connect with you on LinkedIn",
+      "url": "https://www.linkedin.com/in/jessa-challa/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Buy me a coffee... Thank you!",
+      "url": "https://www.buymeacoffee.com/jessachalla",
+      "icon": "FaCoffee"
+    }
+  ],
```
Comment (ChinmayMhatre) [inline data/jessachalla.json:47]:
```suggestion
  ]
```
Please remove this comma

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
A minor change to the json file. Thank you!

**Comment by jessachalla** [comment]
Thank you for your help!

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

File: data/jessachalla.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
```
Comment (Pradumnasaraf) [inline data/jessachalla.json:47]:
```suggestion
  ]
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, 

Please remove the extra comma `,`

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**